### PR TITLE
Deploys Storybook to S3

### PR DESCRIPTION
**Pull request by dzucconi**
Would also be nice to get back PR-specific deploys — are we doing that anywhere else?

### CODE REVIEW

**Comment by zephraph** [comment]
I don't think we're doing PR specific deploys anywhere. We could build that out, but it'd take a bit of finagling.

File: .circleci/config.yml [hunk 1]
```diff
@@ -54,3 +65,24 @@ workflows:
             - lint-changed
             - yarn/type-check
             - yarn/update-cache
+      - yarn/run:
+          name: build-docs
```
Comment (zephraph) [inline .circleci/config.yml:69]:
```suggestion
          name: build-storybook
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -54,3 +65,24 @@ workflows:
...
+                root: .
+                paths:
+                  - storybook_build
+          filters:
+            branches:
+              only: master
+      - upload-docs:
+          context: static-sites-uploader
+          requires:
+            - build-docs
```
Comment (zephraph) [inline .circleci/config.yml:82]:
```suggestion
            - build-storybook
```

not really a necessary change, but it'd be a little more clear

File: .circleci/config.yml (hunk 1 above)
Comment (dzucconi) [reply .circleci/config.yml]: 👍

**Comment by artsyit** [comment]
:rocket: PR was released in [`v26.31.4`](https://github.com/artsy/reaction/releases/tag/v26.31.4) :rocket:

**Pull request merged**